recipe/meta.yaml

```
@@ -1,16 +1,16 @@
-{% set version = "3.9.0" %}
+{% set version = "3.12.0" %}
 
 package:
   name: flit-suite
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/f/flit/flit-{{ version }}.tar.gz
-  sha256: d75edf5eb324da20d53570a6a6f87f51e606eee8384925cd66a90611140844c7
+  url: https://pypi.org/packages/source/f/flit/flit-{{ version }}.tar.gz
+  sha256: 1c80f34dd96992e7758b40423d2809f48f640ca285d0b7821825e50745ec3740
   folder: source
 
 build:
-  number: 1
+  number: 0
   skip: True  # [py<36]
```

https://github.com/pypa/flit/blob/3.12.0/pyproject.toml#L17
```suggestion
  skip: True  # [py<38]
```